src/Tizen.MachineLearning.Inference/Tizen.MachineLearning.Inference/SingleShot.cs

```diff
@@ -179,6 +179,58 @@ public void SetTimeout(int ms)
             NNStreamer.CheckException(ret, "fail to set the timeout!");
         }
 
+        /// <summary>
+        /// Sets the property value for the given model. Note that a model/framework may not support changing the property after opening the model.
+        /// </summary>
+        /// <param name="name">The property name</param>
+        /// <param name="value">The property value</param>
+        /// <feature>http://tizen.org/feature/machine_learning.inference</feature>
+        /// <exception cref="NotSupportedException">Thrown when the feature is not supported.</exception>
+        /// <exception cref="ArgumentException">Thrown when the method failed due to an invalid parameter.</exception>
```

Comment:
I was check native code, it only allow some specific name, so you need to describe it on the API description

If the current number of properties provided by Native, 
it is recommended to make as individual properties.

``` c#
public string Input {get;set;}
public string Intputtype {get;set;}
public string InputName {get;set;}
public bool IsUpdatable {get;set;}
...
```

Reply:
@gichan-jang plz update details.
```
A model/framework may support changing the model information, such as tensor dimension and data layout, after opening the model. If tries to change unavailable property or the model does not allow changing the model information, this will raise an exception.
For the details about the properties, see 'tensor_filter' plugin definition in NNStreamer (https://github.com/nnsuite/nnstreamer).
```

Reply:
Checked. I will update it.